--- a/MODULE.bazel	2026-04-23 21:25:43
+++ b/MODULE.bazel	2026-04-23 21:25:43
@@ -76,26 +76,6 @@
 go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
 
 # to be removed as dependency
-# <<< REMOVE_START >>>
-go_deps.module_override(
-    patch_strip = 1,
-    patches = ["//third_party:com_github_ethereum_go_ethereum_secp256k1.patch"],
-    path = "github.com/ethereum/go-ethereum",
-)
-go_deps.module_override(
-    patch_strip = 1,
-    patches = ["//third_party:com_github_bytedance_sonic.patch"],
-    path = "github.com/bytedance/sonic",
-)
-go_deps.gazelle_override(
-    build_file_generation = "off",
-    path = "github.com/wasmerio/wasmer-go",
-)
-go_deps.gazelle_default_attributes(
-    directives = ["gazelle:proto disable"],
-)
-
-# <<< REMOVE_END >>>
 #
 go_deps.from_file(go_mod = "//:go.mod")
 use_repo(
